### PR TITLE
add sample data help methods

### LINKED ISSUE #11: create helper methods to facilitate sanity test automation task
**Is your feature request related to a problem?**
create helper methods to facilitate sanity test automation task

**What solution would you like?**
create helper methods to facilitate sanity test automation task

### PR DESCRIPTION
**Pull request by Tengda-He**
### Description
create adding sample data help methods in utils
 
### Issues Resolved
#11

### Test

https://user-images.githubusercontent.com/89666799/142688099-36e4108d-2e3e-493d-a0ad-9ae56dd47237.mp4

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: common-utils/misc-utils.js [hunk 1]
```diff
@@ -44,4 +44,46 @@ export class MiscUtils {
     this.testRunner.get('[data-test-subj="queryInput"]').type('{selectall}{backspace}')
     this.testRunner.get('[data-test-subj="querySubmitButton"]').click()
   }
+
+  /**
+   * Add sample data of Sample eCommerce orders, Sample flight data, Sample web logs to dashboard
+   */
+  addSampleData() {
+    this.testRunner.visit('app/home#/tutorial_directory')
+    this.testRunner.get('button[data-test-subj="addSampleDataSetecommerce"]').should('be.visible').click()
```
Comment (seraphjiang) [inline common-utils/misc-utils.js:53]: I would prefer to extract there `'button[data-test-subj="addSampleDataSetecommerce"]'` selector/xpath to a const for readability and maintainability

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opensearch-dashboards-test/opensearch-dashboards-test-library",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (seraphjiang) [inline package.json]: we may start to have version management for this package. when/how to bump up.

**Review by seraphjiang — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Tengda-He) [reply package.json]: This is not part of code change, my bad I don't merge first. it is gone now.

Comment (seraphjiang) [reply package.json]: i guess you mean rebase :)

btw, it is not in scope of PR, but we need version strategy for this package so we could scale while multiple people work on it

Comment (Tengda-He) [reply package.json]: with the package.json in functional-test pointed to test-library git repo, we don't need version update for every test-library change. For my local development, I forked this test-library, and in my functional-test dependency it points to my forked repo.

**Pull request merged**